Model/Ui/AdyenCcConfigProvider.php

```diff
@@ -169,6 +169,14 @@ public function getConfig()
             $config['payment']['adyenCc']['installments'] = [];
         }
 
+        // check if cvc is required
+        $config['payment']['adyenCc']['cvcRequired'] = $this->configHelper->getConfigData(
```

Comment:
@RokPopov would you mind using either (`cvc_required` & `cvcRequired`) or (`require_cvc` & `requireCvc`) throughout? So that we keep the naming consistent with the config path